api/asap.md

```diff
@@ -0,0 +1,50 @@
+# asap_smooth()  <tag type="toolkit">Toolkit</tag><tag type="experimental">Experimental</tag>
+The [ASAP smoothing alogrithm](https://arxiv.org/pdf/1703.00983.pdf) is designed create human readable graphs which preserve the rough shape and larger trends of the input data while minimizing the local variance between points.  The `asap_smooth` hyperfunction provides an implementation of this algorithm which takes `(timestamptz, double precision)` data and returns an ASAP smoothed [`timevector`][hyperfunctions-timevectors].
+
+## Required Arguments
+|Name| Type |Description|
+|---|---|---|
+| `ts` | `TIMESTAMPTZ` | Column of timestamps corresponding to the values to aggregate |
+| `value` | `DOUBLE PRECISION` |  Column to aggregate. |
+| `resolution` | `INT` |  Approximate number of points to return.  Intended to represent the horizontal resolution in which the aggregate will be graphed
+
+## Returns
+
+|Column|Type|Description|
+|---|---|---|
+| `normalizedtimevector` | `NormalizedTimevector` | A object representing a series of values occurring at set intervals from a starting time.  It can be unpacked via `unnest` |
```

Comment:
```suggestion
|Column|Type|Description|
|-|-|-|
|`normalizedtimevector`|`NormalizedTimevector`|An object representing a series of values occurring at set intervals from a starting time. It can be unpacked via `unnest`.|
```